### PR TITLE
add conv3d and pool3d, fix unsqueeze2

### PR DESCRIPTION
**Pull request by yeliang2258**
1. add conv3d and pool3d
2. fix unsqueeze2 OP
related Issue: https://github.com/PaddlePaddle/Paddle2ONNX/issues/404

### CODE REVIEW

**Comment by jiangjiajun** [comment]
@yeliang2258  
The pr lacks of unit test of `conv3d`/`pool3d`/`squeeze2`, and please refer this pull request https://github.com/PaddlePaddle/Paddle2ONNX/pull/400, there are some corner case we didn't test before, more cases should be tested, include all kinds of parameters(`groups`, `dilations`...)

**Comment by yeliang2258** [comment]
> @yeliang2258 The pr lacks of unit test of `conv3d`/`pool3d`/`squeeze2`, and please refer this pull request #400, there are some corner case we didn't test before, more cases should be tested, include all kinds of parameters(`groups`, `dilations`...)

done, add AdaptiveAvgPool2D, AdaptiveAvgPool3D, MaxPool3D test and fix Conv2D, Conv3D

File: paddle2onnx/op_mapper/nn.py [hunk 1]
```diff
@@ -23,25 +23,26 @@
...
         kernel_shape = node.input_shape('Filter', 0)
         dilations = node.attr('dilations')
-        kernel_shape = kernel_shape[-2:]
+        kernel_shape = kernel_shape[2:]
         strides = node.attr('strides')
         group = node.attr('groups')
         pads = node.attr('paddings')
-        assert node.attrs['data_format'] == 'NCHW', "The conv data format should be 'NCHW', but received data format " \
-                                                    "is %s." % node.attrs['data_format']
+
```
Comment (jiangjiajun) [inline paddle2onnx/op_mapper/nn.py]: assert is required,  only support NCHW/NCDHW now

File: paddle2onnx/op_mapper/nn.py [hunk 2]
```diff
@@ -190,6 +191,108 @@ def opset_1(cls, graph, node, **kw):
...
+            start = math.floor(i * (in_size / out_size))
+            end = math.ceil((i + 1) * (in_size / out_size))
+            spans.append(end - start)
+        if len(set(spans)) == 1:
+            return True
+        return False
+
+    @classmethod
+    def opset_1(cls, graph, node, **kw):
+        if node.attr('global_pooling') or (node.attr('adaptive') and
```
Comment (jiangjiajun) [inline paddle2onnx/op_mapper/nn.py:224]: check if pool3d need assert `data_format`

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: paddle2onnx/op_mapper/nn.py (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/op_mapper/nn.py]: done, add assert

File: paddle2onnx/op_mapper/nn.py (hunk 2 above)
Comment (yeliang2258) [reply paddle2onnx/op_mapper/nn.py:224]: done, add data_format assert

**Review by jiangjiajun — APPROVED** [review]
